v3/integration/lints/main.go

```diff
@@ -0,0 +1,77 @@
+package main // import "github.com/zmap/zlint/v3/integration/lints"
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/zmap/zlint/v3/integration/lints/lint"
+
+	"github.com/zmap/zlint/v3/integration/lints/lints"
+)
+
+var Linters = []lint.Lint{
+	&lints.InitFirst{},
+}
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Printf("USAGE ./main <path to lint directory>\n")
```

Comment:
nit: I'd be tempted to use `os.Args[0]` in place of `./main` here.
```suggestion
		fmt.Printf("USAGE ./main <path to lint directory>\n")
```